packages/addressable/rollup.cjs.config.js

```diff
@@ -0,0 +1,59 @@
+import visualizer from 'rollup-plugin-visualizer';
+import typescript from 'rollup-plugin-typescript2';
+import tscompile from 'typescript';
+import filesize from 'rollup-plugin-filesize';
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import pkg from './package.json';
+import babel from 'rollup-plugin-babel';
+
+export default {
+  input: 'src/index.ts',
+  output: [
+    {
+      file: pkg.main,
+      format: 'cjs',
+      sourcemap: false,
+    },
+  ],
+  external: ['rxjs'],
```

Comment:
Why do you need to specify `rxjs` here as you don't use it at all in this package ?